install/1.61-1.62/update.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE `wD_Games`
+ADD COLUMN `nextPhaseMinutes` int(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `phaseMinutes`,
+ADD COLUMN `phaseSwitchPeriod` int(10) DEFAULT -1 AFTER `nextPhaseMinutes`,
+ADD COLUMN `startTime` int(10) UNSIGNED DEFAULT NULL;
+
+UPDATE wD_Games
+SET nextPhaseMinutes = phaseMinutes
+WHERE nextPhaseMinutes = 0;
+
```

Comment:
You need to add an update statement to this file to move the version of the sql file up to match the version in the definitions file otherwise the site code version and sql version won't match upon go live and the site will crash.

Reply:
Done